app/cafe.py

```diff
@@ -0,0 +1,22 @@
+from datetime import date
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError("You must have a vaccine")
+        if visitor["vaccine"]["expiration_date"] < date.today():
+            raise OutdatedVaccineError("You expiration date is out")
```

Comment:
it seems to me, that is better to write here: "vaccine expiration date is out" not "you" :)